src/Context/DrupalErrorCheckApiContext.php

```diff
@@ -14,6 +14,18 @@
  */
 class DrupalErrorCheckApiContext extends RawDrupalContext {
 
+  /**
+   * Watchdog errors severity level.
+   *
+   * @var RfcLogLevel $severity_level
+   */
+  protected static $severity_level;
+
+  public function __construct($severity_level = RfcLogLevel::WARNING)
+  {
+    static::$severity_level = $severity_level;
```

Comment:
variable should not be static, when it's used only non-static as here.

Reply:
It is used in a static function checkForWatchdogErrors(), where it can not be used with $this->

Reply:
ok. I see let's simply comment this then.